### DISCUSSION TITLE
<space> Y not yanking to system clipboard

### DISCUSSION THREAD

**Discussion by BKSalman** [helix-editor/helix#3923] [Q&A] [answered]
so basically ``<space> Y`` is not yanking to system clipboard

I use Nobara Linux ( a derivative of fedora ), and I installed helix through the repo in the installation guide

it used to work before switching from windows to linux

(idk if I should've made this an issue rather than a discussion tbh, sorry)

**Comment by CBenoit** [comment, marked as answer]
You probably need to install a system clipboard provider such as `xclip`.

For starters, try running `helix --health`. This will output you diagnostics about various stuff including system clipboard support.
In my case, I get the following for system clipboard:

```
Clipboard provider: xclip
System clipboard provider: xclip
```

> (idk if I should've made this an issue rather than a discussion tbh, sorry)

No problem, we can open an issue if the need arises. :slightly_smiling_face:

**Reply by BKSalman** [reply]
ah, thanks a lot!

I'm quite new to linux :)

> No problem, we can open an issue if necessity arises. slightly_smiling_face

yep, I  thought it was too basic to be an issue, didn't want to clutter the issues
